docs/questions/native-editor/basic-sql-parameters.md

```diff
@@ -0,0 +1,71 @@
+---
+title: Basic SQL parameters
+summary: Text, number, and date variables let you plug basic values into your SQL code.
+---
+
+# Basic SQL parameters
+
+> If you want to filter on a database field in your query, you should prefer using [field filter variables](./sql-parameters.md), which require a different syntax.
+
+To add a basic variable to a SQL query, enclose the variable in double braces: `{% raw %}{{variable_name}}{% endraw %}`.
+
+This example defines a **Text** variable called `category_filter` (but you can call the variable whatever you want):
+
+```sql
+{% raw %}
+SELECT
+  count(*)
+FROM
+  products
+WHERE
+  category = {{category_filter}}
+{% endraw %}
+```
+
+These basic variables (that just allow basic string/number interpolation) have a different syntax than [field filters](./sql-parameters.md).
```

Comment:
> that just allow basic string/number interpolation

I think this needs less jargon-y explanation (also, as a data point, i knew what string interpolation means, but it was not clear how that concept applies to the variables)